docs/unittesting.md

```diff
@@ -68,9 +130,9 @@ function checkSenderIs0AndValueis10 () public payable {
     Assert.equal(msg.value, 10, "wrong value in checkSenderIs0AndValueis10");
 }
 ```
-Things to keep in mind while using custom transaction context:
+<u>Instructions to use:</u>
 
-1. Parameters must be defined in devdoc of related method
+1. Parameters must be defined in the method natspec
 2. Each parameter key should be prefixed with a hash (**#**) and end with a colon following a space (**:&nbsp;**) like `#sender: ` & `#value: `
 3. For now, customization is available for parameters `sender` & `value` only
 4. Sender is `from` address of a transaction which is accessed using `msg.sender` inside a contract method. It should be defined in a fixed format as '**account-**<account_index>'
```

Comment:
4. The sender is the `from` address of a transaction which is accessed using `msg.sender` inside a contract method.

Reply:
done